test/utility/backup.c

```diff
@@ -274,6 +274,48 @@ __int_comparator(const void *a, const void *b)
     return (*(int *)a - *(int *)b);
 }
 
+/*
+ * testutil_last_backup_id --
+ *     Find the last valid backup id number. Can be used after a restart to determine backups based
+ *     on directory naming.
+ */
+void
+testutil_last_backup_id(int *last_full, int *last_incr)
+{
+#ifdef _WIN32
+    WT_UNUSED(last_full);
+    WT_UNUSED(last_incr);
+#else
```

Comment:
What's our story for WIN32?  If we don't support it, or don't support backup testing on it, we should probably assert here.